src/dpp/presence.cpp

```diff
@@ -254,7 +263,12 @@ std::string presence::build_json(bool with_id) const {
 				{ "type", i.type }
 			});
 			if (!i.url.empty()) j2["url"] = i.url;
-			if (!i.state.empty()) j2["details"] = i.state; // bot activity is details, not state
+
+            if(i.type == activity_type::at_custom) {
```

Comment:
these are space indented, this is why it is not working right.
Please change your editor to only use tab indents with tab size 8. You can then re-indent it

Reply:
Ah is that what that is. Thank you!

Reply:
Indentation has been corrected.